build.gradle

```
@@ -89,7 +91,7 @@ dependencies {
 compileScala {
   scalaCompileOptions.additionalParameters = [
     "-Xplugin:" + configurations.scalaCompilerPlugin.asPath,
-    "-P:scapegoat:dataDir:" + buildDir + "/reports/scapegoat/src/",
+    "-P:scapegoat:dataDir:" + rootDir + "/build/reports/scapegoat/src/",
```

One can use this instead to reference the build directory in gradle (same below):

```suggestion
    "-P:scapegoat:dataDir:" + layout.buildDirectory.dir("reports/scapegoat/src/"),
```

---

Then nevermind